chatbot-ui/pages/index.tsx

```diff
@@ -30,6 +43,9 @@ export const getServerSideProps = (async () => {
   return {
     props: {
       apiUrl: process.env.CHAT_URL_API ?? 'http://localhost:8010/api/chat',
+      // Could put this in KeyVault, but probably sufficient for now as an urgent stop-gap just for today?
+      chatbotPassword:
+        process.env.AUTH_PASSWORD ?? 'fb1a3bb9-7b35-47e2-8a45-44db18215912',
```

Comment:
You could add this to `.env.example`. with a default value.

You could also add this to `main.parameters.json`:

```
    "webAppDefinition": {
      "value": {
        "settings": [
          {
            "name": "AUTH_PASSWORD ",
            "value": "${AUTH_PASSWORD}"
          }
        ]
      }
    }
```

and set this as a variable in the pipeline config in Azure DevOps. Then we can remove this value here. Later we can work on changing it to be a secret.